src/app/components/shared/menu/link/link.component.ts

```diff
@@ -109,13 +109,8 @@ export class MenuLinkComponent implements OnChanges {
 
   private handleExternalLink(): void {
     const uri = this.externalLink.uri(this.activatedRoute.snapshot.params);
-
-    /*
-     * TODO This is a potential future point of failure if the website and the api
-     * do not exist on the same domain
-     *
-     * Prepend apiRoot to relative links
-     */
+    // Redirect relative routes to api
+    // ! This seems unintuitive and likely needs to be changed in the future
     this.href = uri.startsWith("/") ? this.apiRoot + uri : uri;
```

Comment:
agreed this is dodgy. We should probably ensure an url returned by an API object gets transformed before it gets to the model so that there are no relative links